fetch-in-react/my-app/src/Users.jsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import UserDetails from './UserDetails';
+import UserCard from './UserCard';
+import './Users.css';
+
+export default function Users() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+  const [users, setUsers] = useState([]);
+  const [user, setUser] = useState();
+
+  /* your code here (hint: useEffect) */
+  useEffect(() => {
+    async function fetchUsers() {
+      try {
+        const res = await fetch('https://jsonplaceholder.typicode.com/users');
+        if (!res.ok) {
+          throw new Error(`Error: , status code: ${res.status}`);
+        }
+        const resUsers = await res.json();
+        setUsers(resUsers);
+        setIsLoading(false);
+      } catch (error) {
+        setError(error);
+        setIsLoading(false);
+      }
+    }
+    fetchUsers();
+  }, [users]);
```

Comment:
This isn't quite what you want. `users` is not a dependency in the effect function. You call `setUsers` which changes `users` indirectly, but does not use it directly. Your dependencies array should list things that you use directly.

So the correct dependency array is just an empty array. (As you answered in Q&A! ;-) )

Reply:
Okay, got it! Thanks for the advice!